.kitchen.ci.yml

```diff
@@ -0,0 +1,48 @@
+---
+driver:
+  name: rackspace
+  rackspace_username: <%= ENV['RACKSPACE_USERNAME'] %>
+  rackspace_api_key: <%= ENV['RACKSPACE_API_KEY'] %>
+  rackspace_region: 'lon'
+  flavor_id: 'performance1-1'
+  privileged: true
+  require_chef_omnibus: 12.7.2
+
+provisioner:
+  name: chef_zero
+  data_path: test/shared
+
+platforms:
+  - name: ubuntu-14.04
+    run_list:
+      - recipe[apt]
+
+suites:
+  - name: postgresql
+    data_bags_path: "test/integration/mysql/data_bags"
+    encrypted_data_bag_secret_key_path: "test/integration/mysql/data_bag_key"
+    run_list:
+      - recipe[iptables]
```

Comment:
Oh, it's only for the CI test suite and not used anywhere else. Anyway, above statement still stands :-)